src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__month-length--31 calendar__start-day--mon">
+      <div class="cell"></div>
```

Comment:
Maybe, it`s not BEM block. Better to use calendar__cell.

Reply:
Agree, fixed